src/main/java/sx/blah/discord/api/internal/DiscordWS.java

```diff
@@ -47,6 +48,11 @@
 
 public class DiscordWS extends WebSocketAdapter {
 
+	/**
+	 * The version of the Discord Gateway that is used by Discord4J.
+	 */
+	public static final String GATEWAY_VERSION = "6";
```

Comment:
I feel like this is redundant. Is there a good reason to maintain two separate variables which have the same value?